Toggl.Daneel/ViewControllers/SelectClientViewController.cs

```diff
@@ -20,41 +29,32 @@ public override void ViewDidLoad()
         {
             base.ViewDidLoad();
 
-            var source = new ClientTableViewSource(SuggestionsTableView);
-            SuggestionsTableView.Source = source;
-
-            var bindingSet = this.CreateBindingSet<SelectClientViewController, SelectClientViewModel>();
-
-            bindingSet.Bind(source).To(vm => vm.Suggestions);
-            bindingSet.Bind(SearchTextField).To(vm => vm.Text);
-            bindingSet.Bind(CloseButton).To(vm => vm.CloseCommand);
-            bindingSet.Bind(source)
-                      .For(v => v.SelectionChangedCommand)
-                      .To(vm => vm.SelectClientCommand);
-
-            bindingSet.Bind(source)
-                      .For(v => v.CreateClientCommand)
-                      .To(vm => vm.CreateClientCommand);
+            SuggestionsTableView.RegisterNibForCellReuse(ClientViewCell.Nib, ClientViewCell.Identifier);
+            SuggestionsTableView.RegisterNibForCellReuse(CreateClientViewCell.Nib, CreateClientViewCell.Identifier);
+            SuggestionsTableView.SeparatorStyle = UITableViewCellSeparatorStyle.None;
+            SuggestionsTableView.Source = tableViewSource;
 
-            bindingSet.Bind(source)
-                      .For(v => v.SuggestCreation)
-                      .To(vm => vm.SuggestCreation);
+            ViewModel.Clients.Subscribe(replaceClients).DisposedBy(DisposeBag);
```

Comment:
I think this is better in three lines.
```suggestion
            ViewModel.Clients
                .Subscribe(replaceClients)
                .DisposedBy(DisposeBag);
```